src/components/contact-datalist-item/ContactDatalistItem.js

```diff
@@ -7,17 +7,25 @@ import DatalistItem from '../datalist-item';
 import './ContactDatalistItem.scss';
 
 type Props = {
-    id?: string | number | null,
+    getContactAvatarUrl?: (contact: { id: string | number, [key: string]: any }) => string,
```

Comment:
including the only required field and allowing the rest to vary for now. if we had a top-level type matching contact data, we could support a stricter type here.